policy/pa_test.go

```diff
@@ -153,15 +169,17 @@ func TestWillingToIssue(t *testing.T) {
 		`banned.in.dc.com`,
 	}
 
-	shouldBeAccepted := []string{
-		`lowvalue.website1.org`,
-		`website4.sucks`,
-		"www.unrelated.com",
-		"unrelated.com",
-		"www.8675309.com",
-		"8675309.com",
-		"web5ite2.com",
-		"www.web-site2.com",
+	shouldBeAccepted := identifier.ACMEIdentifiers{
+		identifier.NewDNS(`lowvalue.website1.org`),
+		identifier.NewDNS(`website4.sucks`),
+		identifier.NewDNS(`www.unrelated.com`),
+		identifier.NewDNS(`unrelated.com`),
+		identifier.NewDNS(`www.8675309.com`),
+		identifier.NewDNS(`8675309.com`),
+		identifier.NewDNS(`web5ite2.com`),
+		identifier.NewDNS(`www.web-site2.com`),
+		identifier.NewIP(netip.MustParseAddr(`9.9.9.9`)),
+		identifier.NewIP(netip.MustParseAddr(`2620:fe::fe`)),
```

Comment:
IIRC, RFC 8738 ACME IP says that the IP must be presented in the [RFC 5952 recommended format](https://datatracker.ietf.org/doc/html/rfc5952#autoid-28). Should our tests include IPv6 addresses that are _not_ in that format, and ensure that we reject them?

Reply:
Good catch! I think tests for this belong in `TestWellFormedIdentifiers` above, though, not `TestWillingToIssue` - since `WillingToIssue` is only doing wildcard and blocklist checks.